### PR TITLE
chore: some ui improvements

### LINKED ISSUE #2761: [Bug]
### Bug Description

In the card view, the property action menu needs design polish
image

### How to Reproduce

Create a card on a board
Open it up
Click on a property name

### Expected Behavior

More polished

### Operating System

Windows 11

### AppFlowy Version(s)

v0.2.0

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by richardshiue**
A couple of UI improvements in the grid UI:

- Remove hack for https://github.com/flutter/flutter/issues/18761 . Due to the removal of the hack that adds a zero-width space before every single character, the integration test will now find more widgets with the text "Read me", presumably in the sidebar where FlowyText is used. Therefore, I'm changing this findOneWidget to findWidgets.
- Add icons for hide and delete field
  image
- Slightly bump up the height of select option editor to make everything in the popup fit by default
- Fixed padding on date and time format buttons

fixes #2761

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/select_option.dart [hunk 1]
```diff
@@ -209,7 +209,7 @@ class _OptionCellState extends State<_OptionCell> {
       offset: const Offset(8, 0),
       margin: EdgeInsets.zero,
       asBarrier: true,
-      constraints: BoxConstraints.loose(const Size(460, 460)),
+      constraints: BoxConstraints.loose(const Size(460, 470)),
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/select_option.dart:212]: I'm curious, why this change?

**Comment by appflowy** [comment]
I created a integration test for creating a option in single select cell. Maybe you can try to add more tests.

https://github.com/AppFlowy-IO/AppFlowy/assets/86001920/ac078424-0908-4e3b-887b-1ae059192117

File: frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/select_option.dart (hunk 1 above)
Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/select_option.dart:212]: I want to make everything in the select option editor popup fit by default.

**Comment by richardshiue** [comment]
> I created a integration test for creating a option in single select cell. Maybe you can try to add more tests.
> create_option.mp4.mov

Of course!
